lazy_tensor_core/lazy_tensor_core/csrc/ops/generic.h

```diff
@@ -33,6 +33,38 @@ class Generic : public torch::lazy::TsNode {
   torch::lazy::hash_t hash_seed_;
 };
 
+inline torch::lazy::NodePtr GenericOp(
```

Comment:
Do we need all of the helpers? And you probably don't need inline as the function is defined in the header.

Reply:
which helpers? you mean do we need GenericOp anymore?

Reply:
We have 4 overloads here. I'm not sure if we actually need all of them.

Reply:
Initially PyTorch/XLA uses all the helpers here, but you are right, after switching to codegen, we don't need all of them.

Reply:
Not sure I understand your comment about`inline`. Being defined in the header doesn't guarantee a function will be inlined, right?

Reply:
There is no guaranteed `inline` as far as you know. The compiler has the free will to make the final call. Implicit `inline` in headers should be enough.

Reply:
I see `inline` is used in core almost as a coding convention, so I guess I would prefer to leave it as is.

Reply:
Hmm, up to you. We don't have any coding styles for reference anyway. I'm not going to force you. 😃